src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -135,7 +143,8 @@ private void UpdateTree()
 
             if (viewModelCount > 0)
             {
-                listControl.lvDetails.SelectedItem = 0;
+                nonFrameworkListControl.lvDetails.SelectedItem = 0;
+                frameworkListControl.lvDetails.SelectedItem = 0;
                 this.spHowToFix.DataContext = itemViewModel.First<ScanListViewItemViewModel>();
```

Comment:
itemViewModel could be in either list. We probably want it to be from nonFrameworkIssues if we have any, otherwise frameworkIssues. It's important that the selection and the HowToFix are in sync

Reply:
If we just delete this line the first time an item is selected, the HowToFix section is not updated, so I added the corresponding correspondent setting for DataContext.